runtime/src/accounts_db.rs

```diff
@@ -552,14 +551,8 @@ impl AccountsDB {
                         AppendVec::new_relative_path(slot_id, storage_entry.id);
                     let append_vec_abs_path =
                         append_vecs_path.as_ref().join(&append_vec_relative_path);
-                    let mut copy_options = CopyOptions::new();
-                    copy_options.overwrite = true;
-                    let e = fs_extra::move_items(
-                        &vec![&append_vec_abs_path],
-                        &local_dir,
-                        &copy_options,
-                    )
-                    .map_err(|e| {
+                    let target = local_dir.join(append_vec_abs_path.file_name().unwrap());
+                    let e = std::fs::rename(append_vec_abs_path.clone(), target).map_err(|e| {
```

Comment:
Will fs::rename fall back to a copy if the source and destination are on different file systems?

Reply:
Looks like no, from the docs:
> This will not work if the new name is on a different mount point.
Maybe we can rename and if that fails then try a copy